### PR TITLE
Revert LPS-158350 Fix integration tests

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-173188

### CODE REVIEW

File: modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java [hunk 1]
```diff
@@ -496,9 +496,9 @@ public void testPostDataDefinitionByContentType() throws Exception {
 		catch (Problem.ProblemException problemException) {
 			Problem problem = problemException.getProblem();
 
-			Assert.assertNull(problem.getDetail());
 			Assert.assertEquals("BAD_REQUEST", problem.getStatus());
 			Assert.assertEquals("MustSetValidType", problem.getType());
+			Assert.assertEquals("string", problem.getDetail());
```
Comment (carolmariaabb) [inline modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java:501]: @marcelabc I would change this to the same order as the others, i.e., `problem.getDetail()` before `problem.getStatus()`

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**